README.md

```diff
@@ -15,9 +15,5 @@ Syntax:
 |[Kartik Gajjar](https://twitter.com/Kartikt2twt)| Happy Birthday Eddie..Lots of lovee |
 |[Dunsin](https://twitter.com/DunsinWebDev)|Happy Birthday Eddie, have a great and wonderful day|
 |[Pranshu Kumar Agrawal](https://twitter.com/pkknowsnothing)| Happiest birthday Eddie Sir. May all your wishes come true.|
```

Comment:
```suggestion
|[Pranshu Kumar Agrawal](https://twitter.com/pkknowsnothing)| Happiest birthday Eddie Sir. May all your wishes come true.|
|[Krupali Trivedi](https://twitter.com/chai_really)|Happy Birthday Eddie! Thank you for the awesome community. Wishing happy happy years ahead!|
|[Aarya Chopkar](https://twitter.com/ChopkarAarya)| Happy birthday. Keep inspiring, continue your amazing work.|
|[Aditya Joshi](https://twitter.com/aditya__exe) | Happy Birthday Eddie! Loads of love❤️|
|[Pushkaraj Kulkarni](https://twitter.com/pushkaraj2007)|Happy Birthday Eddie! Have a great year ahead, and Thanks for motivating us to start open source journey.|
|[Shubhangi](https://twitter.com/mshubhangi12)| Happy Birthday, Eddie! On your birthday, On your birthday, I wish you a lifetime of good health so that you can continue to inspire us and guide us during tumultuous times!! :heart_eyes: :balloon: :confetti_ball: :tada::sparkles:
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix